vm/systemSmartContracts/staking_test.go

```diff
@@ -0,0 +1 @@
+package systemSmartContracts
```

Comment:
missing tests. Might be left to a next PR after we decide this is the structure that we want.

Reply:
wanted to leave for the next PR as code is a lot.

Reply:
and also might give the writing of unit tests to a junior.